src/connectors/chillhop.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+Connector.playerSelector = '#player-controlls';
```

Comment:
This isn't a typo, it's how it is in the page.

Reply:
Could you add a comment about this in the code?

Reply:
Done.